x-pack/filebeat/module/okta/system/test/okta-system-test.json.log-expected.json

```diff
@@ -0,0 +1,251 @@
+[
+    {
+        "@timestamp": "2020-02-14T22:18:51.843Z",
+        "client.as.number": null,
+        "client.as.organization.name": null,
+        "client.domain": null,
+        "client.geo.city_name": "Dublin",
+        "client.geo.country_name": "United States",
+        "client.geo.location.lat": 37.7201,
+        "client.geo.location.lon": -121.919,
+        "client.geo.region_name": "California",
+        "client.ip": "108.255.197.247",
+        "event.action": "user.session.end",
+        "event.category": "authentication",
+        "event.dataset": "okta.system",
+        "event.id": "faf7398a-4f77-11ea-97fb-5925e98228bd",
+        "event.kind": "event",
+        "event.module": "okta",
+        "event.outcome": "SUCCESS",
+        "event.type": "access",
```

Comment:
should be an array

Reply:
Setting this value to be a string seems to be the common practice in all our modules today, e.g. googlecloud, aws, azure, etc.